src/main/resources/component-biz.conf

```
@@ -1418,7 +1425,7 @@ security {
         #The 'tenant' scope used by the tenants framework (TenantUserManager etc.)
         tenant {
             # the packages of the scope
-            packages = []
+            packages = ["logged-in"]
```

```suggestion
            packages = []
```
Think this change is not required